### ISSUE TITLE
Add the package DC++ PtokaX 

### ISSUE THREAD

**Issue by dcbeelinekz** [openwrt/packages#5460]
ready-made **server** package **DC++**
https://github.com/dcbeelinekz/ptokax_lede/tree/master/ptokax

**Comment by karlp** [comment]
If it's already a ready package makefile, you'll get more mileage making a proper pull request for it.

**Comment by dcbeelinekz** [comment]
Yes. it is a ready and tested server. Almost a year stands on the router
